sim/paladin/item_sets_pve.go

```diff
@@ -215,24 +216,14 @@ var ItemSetRadiantJudgement = core.NewItemSet(core.ItemSet{
 				Label: "S03 - Item - T2 - Paladin - Retribution 4P Bonus",
 				OnInit: func(aura *core.Aura, sim *core.Simulation) {
 
-					originalApplyEffects := paladin.judgement.ApplyEffects
-
-					// Wrap the apply Judgement ApplyEffects with more Effects
-					paladin.judgement.ApplyEffects = func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-
-						sealsLastJudgement := paladin.lastJudgement // Get Last Judgement Seals before it gets overwritten in originalApplyEffects
-						originalApplyEffects(sim, target, spell)
-						sealsThisJudgement := paladin.lastJudgement // Get Active Seals this Judgement (after lastJudgement is set)
+					newCD := core.Cooldown{
+						Timer:    paladin.NewTimer(),
+						Duration: paladin.judgement.CD.Duration - (5 * time.Second),
+					}
 
-						// Two of the possible implementations - TODO figure out actual implementation
-						// anySealsDifferent := sealsThisJudgement != sealsLastJudgement // Would be nice!
-						// allSealsDifferent := (int(sealsThisJudgement) & int(sealsLastJudgement)) == 0 // More conservative option
-						anySealsGained := (int(sealsThisJudgement) & ^int(sealsLastJudgement)) > 0 // More conservative option (most likely option)
+					paladin.judgement.CD = newCD
```

Comment:
If it just reduces the cooldown by 5 seconds I think you could just do
```
paladin.judgement.CD.Duration -= time.Second * 5
```

Reply:
changed and tested